src/components/Dropdown/Dropdown.types.ts

```diff
@@ -3,10 +3,16 @@ import type {TreeViewData} from '~/components/TreeView/TreeView.types';
 
 type DropdownVariant = 'ghost' | 'outlined';
 
-type DropdownSize = 'small' | 'medium';
+export type DropdownSize = 'small' | 'medium';
```

Comment:
If you use' size="medium" ' instead of `size={'medium' as DropdownSize}`, you shouldn't need to export this type.